src/components/App.js

```diff
@@ -7,7 +7,7 @@ import '../css/App.css';
 
 const stats = require('../stats.json');
 const years = Object.keys(stats).map(k => k.substring(0, 4)).reduce((p, c) => {
-  if (!p.includes(c)) p.push(c);
+  if (p.indexOf(c) === -1) p.push(c);
```

Comment:
Thanks @eoin, odd indeed! Also hi 👋, and nice to hear from you.

I don't understand how that polyfill did not work / was missing but I did witness the exception on a genuine Windows box running IE.

Sounds like somebody has a bug somewhere. Entirely possible it is mine!